napari/util/list/_typed.py

```diff
@@ -6,7 +6,8 @@ def cpprint(obj):
 
 
 class TypedList(List):
-    """Enforce list elements to be of a specific type and allow indexing with their unique properties.
+    """Enforce list elements to be of a specific type and allow indexing with
+    their unique properties.
```

Comment:
This violates PEP257 (one-liner should be one line :joy:), but we can leave that for a later PR, as there's lots of those to clean up.

Reply:
yeah, that's a bit of a pickle isn't it!